libs/server/PubSub/SubscribeBroker.cs

```diff
@@ -408,6 +410,164 @@ public unsafe void Publish(byte* key, byte* value, int valueLength, bool ascii =
             log.Enqueue(logEntryBytes);
         }
 
+        /// <summary>
+        /// Retrieves the collection of channels that have active subscriptions.
+        /// </summary>
+        /// <returns>The collection of channels.</returns>
+        public unsafe void Channels(ref ObjectInput input, ref SpanByteAndMemory output)
+        {
+            var isMemory = false;
+            MemoryHandle ptrHandle = default;
+            var ptr = output.SpanByte.ToPointer();
+
+            var curr = ptr;
+            var end = curr + output.Length;
+
+            try
+            {
+                if (subscriptions is null || subscriptions.Keys.Count == 0)
```

Comment:
```suggestion
                if (subscriptions?.Count == 0)
```

Reply:
For now, I am changing `subscriptions.Keys.Count` to `subscriptions.Count` as marking the comment as resolved